src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -114,16 +114,37 @@
 
       @end
       function {@class.initClientFuncName}() {
+        @if class.auth.type != "NONE"
+          var API_KEY = '';  // TODO: Update placeholder with desired API key.
```

Comment:
It was a recommendation from Bochun, who said that API key should included in the JS samples for both 3LO and API key based auth.